src/index.html

```diff
@@ -1,11 +1,11 @@
-﻿﻿﻿<!DOCTYPE HTML>
+﻿﻿<!DOCTYPE HTML>
 <html lang="en" ng-app="OpenFinD3FC">
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>OpenFin D3FC</title>
-    <link href="assets/css/style.css" rel="stylesheet" >
+    <link href="assets/css/style.css" rel="stylesheet">
     <link href="assets/css/d3fc.min.css" rel="stylesheet" />
     <link href="assets/css/jquery.mCustomScrollbar.css" rel="stylesheet" />
     <style>
```

Comment:
Since you are changing the way this <link> closes, do you want to remove other self closing links? This is the html5 practice.